docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -35,70 +19,36 @@ All occurrences of the code before the `>>` must be substituted with the code af
 - `="conditionType" count="XOR"` >> `="conditionType" count="1"`
```

Comment:
To keep everything listed similarly, I would move this and the following line after the line
- `="conditionType" value="XOR"` >> `="conditionType" value="1"` (I would prefer making two different lists, but this is not related to this update).